runtime/compiler/aarch64/env/J9CPU.cpp

```diff
@@ -39,7 +39,7 @@ CPU::getProcessorFeatureFlags()
 bool
 CPU::isCompatible(TR_Processor processorSignature, TR_ProcessorFeatureFlags processorFeatureFlags)
    {
-   return self()->is(processorSignature);
+   return self()->id() == processorSignature;
```

Comment:
So compatibility requires a perfect match?

Reply:
This is aarch64 and that's how they implemented the compatibility.

Reply:
What we are doing here is just to remove the old `is()` api. The compatibility functions will be re-written in a separate PR. 
The reason these we can't just change these compatibility functions to use the new API now is because this compatibility function is shared across all the platforms. Once all the platforms have the new api's then we can proceed to make them use the new api's.

 `self()->is(processorSignature)` is equivalent to `self()->id() == processorSignature` and that's the old api.